### ISSUE TITLE
"git push heroku master" on WheezyX errors out with "...PRIV_END: seteuid: Permission denied..."

### ISSUE THREAD

**Issue by drhuffman12** [corbinlc/gnuroot#5]
When using git on wheezyX, I get an error when I try to use git to push files, as shown below:

```
root@localhost:/local-dev/ror4/ror4s4# git commit -m "heroku apps:create"
# On branch master
nothing to commit (working directory clean)
root@localhost:/local-dev/ror4/ror4s4# git push heroku master
PRIV_END: seteuid: Permission denied
fatal: The remote end hung up unexpectedly
root@localhost:/local-dev/ror4/ror4s4#
```

**Comment by cedric-vincent** [comment]
For the record, this issue sounds similar to #4

**Comment by cedric-vincent** [comment]
@corbinlc, do you know what is the version of PRoot used by GNURoot WheezyX?

**Comment by cedric-vincent** [comment]
@drhuffman12, does this issue also happen without faking the "root" id? (@corbinlc, I don't know if the option "Fake Root" is still there in GNURoot WheezyX)

**Comment by drhuffman12** [comment]
Yes, I was using 'Launch as Fake Root' option for wheezyx. I was in a sub-folder that I added to the filesystem that gnuroot/wheezyx created. I was able to 'git init', 'git add .', and 'git commit -m "...", but the 'git push heroku master' errored out. (Trying to use git in a Rails folder on my extSdCard errored out sooner during 'git add .', but I think this is probably due to file path length issues when using 'git' in a Rails app under '/host-rootfs/storage/extSdCard/Android/obb/champion.gnuroot.wheezy'... thanks to the security changes added by os 4.4.2 updates to my s4.)

-Daniel

Sent from Yahoo Mail on Android

**Comment by corbinlc** [comment]
Sorry it has been so long.  Can you update to the latest gnuroot and gnuroot wheezy (or wheezyX or wheezy x86, based on your needs) test apks found here:  https://www.dropbox.com/sh/xjbg1z49ur3t5b6/AAAoBzmCK3A27Sk9DZAeolLla?dl=0
This uses proot v5.0.  Can you then try again and tell me what you see?

**Comment by drhuffman12** [comment]
How do I use the wheezy apk's from the dropbox link? I uninstalled the old apk's and I installed the gnuroot20 and wheezy[x] apk's from your dropbox link. The new gnuroot launches, but when I try to create a new rootfs with the new wheezyx, I get an error about "Download failed because resources can not be found...".

See attached screenshots.

Thanks,

Daniel Huffman
864-567-6651
drhuffman12@yahoo.com

**Comment by corbinlc** [comment]
Ah sorry.  The dropbox links will only, easily, work as updates if you have the latest versions installed.  So, please uninstall gruroot wheezy (but keep gruroot, that is fine) then install gruroot wheezy from the store (doing this grabs some extension files that have not changed, but are needed). Then install the version from dropbox.  

There is a slightly more complicated way where you can manually install the .obb extension files, but lets not go there.

**Comment by drhuffman12** [comment]
It looks like 'git push heroku master' works in wheezy and wheezyX. I'll check wheezy86.
